src/commands/new.ts

```diff
@@ -180,7 +180,7 @@ module.exports = {
 
     const yname = boolFlag(options.y) || boolFlag(options.yes)
     const noTimeout = options.noTimeout ?? false
-    const useDefault = (option: unknown) => yname && option === undefined
```

Comment:
The linter was complaining about the name of this function and that it should be a react hook because it begins with "use...". Renamed to `getDefault`.